samples/bookinfo/src/productpage/productpage.py

```diff
@@ -377,6 +377,9 @@ def getProductRatings(product_id, headers):
         request_result_counter.labels(destination_app='ratings', response_code=status).inc()
         return status, {'error': 'Sorry, product ratings are currently unavailable for this book.'}
 
+def send_request(url, **kwargs):
+    # We intentionally do not pool so that we can easily test load distribution across many versions of our backends
+    return requests.get(url, **kwargs)
 
```

Comment:
```suggestion


def send_request(url, **kwargs):
    # We intentionally do not pool so that we can easily test load distribution across many versions of our backends
    return requests.get(url, **kwargs)


```

Reply:
lint wants two blank lines either side of a function